tests/unittests/Foundation/ReferenceFoundation/TestNSURL.mm

```diff
@@ -119,6 +118,8 @@ static boolean setup_test_paths() {
                       gRelativeOffsetFromBaseCurrentWorkingDirectory = strlen(cwdURL.fileSystemRepresentation) + 1;
                   });
 
+// File system functions/constants differ between platforms
+#if TARGET_OS_WIN32
```

Comment:
Yes, but there's enough difference between the constants, as well as defaults on both OSs, that I think anyone writing something like this ought to take the time to look at the filesystem differences manually. (I don't feel comfortable hiding S_IRWXU behind a shim)

---
In reply to: [72315339](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72315339) [](ancestors = 72315339)